File: Practice/SampleActionBar1/app/src/main/java/org/techtown/actionbar/MainActivity.java
package org.techtown.actionbar;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;

import android.app.Activity;
import android.os.Bundle;
import android.view.Menu;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    ActionBar abar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        abar = getSupportActionBar();

        Button button = findViewById(R.id.button1);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                abar.setLogo(R.drawable.home);
                abar.setDisplayOptions(ActionBar.DISPLAY_SHOW_HOME|ActionBar.DISPLAY_USE_LOGO);
            }
        });
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        return super.onCreateOptionsMenu(menu);
    }
}

File: Practice/AndroidStudioProject-master/MyApplication/app/src/main/java/shuworld/com/MainActivity.kt
package shuworld.com

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import shuworld.com.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private var mbinding: ActivityMainBinding? = null
    private val binding get() = mbinding

    override fun onCreate(savedInstanceState: Bundle?) { // 앱이 최초로 실행되었을 때 수행.
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main) // xml 화면 뷰를 연결
    }
}